central/complianceoperator/v2/scanconfigurations/service/convert.go

```diff
@@ -0,0 +1,235 @@
+package service
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/cluster/datastore"
+	complianceDS "github.com/stackrox/rox/central/complianceoperator/v2/scanconfigurations/datastore"
+	v2 "github.com/stackrox/rox/generated/api/v2"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/grpc/authn"
+	"github.com/stackrox/rox/pkg/uuid"
+)
+
+/*
+storage type to apiV2 type conversions
+*/
+
+var (
+	v2IntervalTypeToStorage = map[v2.Schedule_IntervalType]storage.Schedule_IntervalType{
+		v2.Schedule_UNSET:   storage.Schedule_UNSET,
+		v2.Schedule_WEEKLY:  storage.Schedule_WEEKLY,
+		v2.Schedule_MONTHLY: storage.Schedule_MONTHLY,
+	}
+
+	storageIntervalTypeToV2 = map[storage.Schedule_IntervalType]v2.Schedule_IntervalType{
+		storage.Schedule_UNSET:   v2.Schedule_UNSET,
+		storage.Schedule_DAILY:   v2.Schedule_UNSET,
```

Comment:
Is daily not supported at API?

Reply:
I was able to add Daily support with no trouble.